modules/apps/site-initializer/site-initializer-cms/src/main/resources/site-initializer/layouts/07_home/page-definition.json

```diff
@@ -0,0 +1,14 @@
+{
+	"pageElement": {
+		"pageElements": [
+
```

Comment:
@cdorado28 should we create an empty JSP fragment(java class definition for the fragment), similar to how Dashboard page defines:

			{
				"definition": {
					"fragment": {
						"key": "com.liferay.site.cms.site.initializer.internal.fragment.renderer.ViewDashboardJSPSectionFragmentRenderer"
					},
					"fragmentConfig": {
					},
					"fragmentFields": [
					],
					"indexed": true
				},
				"id": "c6aa5b3f-06d7-49ae-a482-baabc9e33daa",
				"type": "Fragment"
			}